test/jdk/javax/swing/JComboBox/DisabledComboBoxFontTestAuto.java

```diff
@@ -44,13 +37,22 @@
 
 import static java.awt.image.BufferedImage.TYPE_INT_ARGB;
 
+/*
```

Comment:
Usually we keep jtreg comment section before imports.

Reply:
There was a discussion going on regarding placement of jtreg tags here https://corparch-core-srv.slack.com/archives/CNQ8SN9P1/p1707238824764849. We updated jtreg tags aftre imports section in recent test sprint.

So, kept it below imports.